contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
In draft of guidelines for Go packaging there is also mentioned preferred way of handling dependencies, for example: `BuildRequires:  golang(github.com/gorilla/mux) >= 0-0.13` instead of `vendor` directory which means separate package for each dep but this is to much (at least for now ;))